pretty_gpx/city/city_drawing_figure.py

```diff
@@ -7,6 +7,7 @@
 
```

Comment:
Thanks @JulesL2 !

The river looks great and pretty accurate

However the query of green areas looks a bit random and not very exhaustive
Maybe the river part can be merged first so that you can rework the park part

![image](https://github.com/user-attachments/assets/007ce388-ec6a-47fc-9232-68af5e1323f2)

Reply:
The "parc de sceaux" could be better handled, as a whole entity
![image](https://github.com/user-attachments/assets/e38d8cc5-681b-4acb-bb00-5a0d1b82bae2)